src/blob-store/index.js

```diff
@@ -204,6 +211,19 @@ export class BlobStore {
     const entry = await drive.entry(path, options)
     return entry
   }
+
+  /**
+   * @param {BlobId} blobId
+   * @param {object} [options]
+   * @param {boolean} [options.diff=false] Enable to return an object with a `block` property with number of bytes removed
+   * @return {Promise<any>}
```

Comment:
any specific reason for using `any` here? may be able to omit this line entirely and rely on inference in this case, since `drive.clear` goes about 3 levels of deps deep 😅

Reply:
ah I see that you added the type for the `clear()` method in this PR, so guessing the `any` was a placeholder?